.github/workflows/algolia-scraper.yml

```diff
@@ -0,0 +1,25 @@
+name: algolia-scraper
+on:
+  push: # only run on push events
+    branches:
+      - main
+  check_suite: # only run check suite events
+    types: [completed]
+  workflow_dispatch: # only run on workflow dispatch events
+jobs:
+  scrape:
+    runs-on: ubuntu-latest # specifies the OS image to use for the job
+    steps: # specifies the commands to run
+      - name: check out code 🛎
+        uses: actions/checkout@v2
+      - name: scrape the site 🧽
+        env: #- specifies the environment variables
+          APPLICATION_ID: ${{ secrets.APPLICATION_ID }} #generated when you create a project on algolia website after you sign up
```

Comment:
In general, prefer putting comments above vs. beside your code, as it lengthens the line length unnecessarily.